app/models/grda_warehouse/tasks/service_history/enrollment.rb

```diff
@@ -21,13 +21,23 @@ def self.batch_process_unprocessed!(max_wait_seconds: 21_600)
         wait_for_processing(max_wait_seconds: max_wait_seconds)
     end
 
+    scope :unassigned, -> do
+      jobs = Delayed::Job.where(queue: :long_running, failed_at: nil).
```

Comment:
Ask ServiceHistory::RebuildEnrollmentsJob which queue it uses, and update here and in `GrdaWarehouse::Tasks::ServiceHistory::Base.wait_for_processing`